lib/puppet/util/plist.rb

```diff
@@ -0,0 +1,100 @@
+require 'cfpropertylist'
```

Comment:
Facter may not always have a dependency on CFPropertyList. The Gemfile should explicitly include it.